components/Tabs/TabLabel.tsx

```diff
@@ -0,0 +1,43 @@
+import cn from "classnames";
+import HeadlessButton from "components/HeadlessButton";
+import { TabsLabel, TabLabelsListProps } from "./types";
+import styles from "./TabLabel.module.css";
+
+const TabLabel = ({ selected, label, onClick }: TabsLabel) => {
+  return (
+    <HeadlessButton
+      disabled={selected}
+      onClick={() => onClick(label)}
+      className={cn(styles.label, selected ? styles.selected : styles.default)}
+    >
+      {label}
+    </HeadlessButton>
+  );
+};
+
+export const TabLabelsList = ({
+  visibleTabs,
+  tabsMeta,
+  currentTab,
+  onClick,
+}: TabLabelsListProps) => {
+  return (
+    <ul
+      className={cn(
+        styles.header,
+        visibleTabs.length ? styles["header-shadow"] : null
+      )}
```

Comment:
I just noticed you are using the `classnames` package, which is great! I was going to suggest that, as it removes the need for ternary here. Try this instead:
```ts
className={cn(
    styles.header,
    {[styles["header-shadow"]]: visibleTabs.length}
  )}
```


Quick note: `cx` is the convention for `classnames`, rather than `cn`.

See https://www.npmjs.com/package/classnames for more info.

Reply:
If you look through out all the repositories on the githab, `cn` is oftener occurrence:
https://github.com/search?q=%22import+cn+from%22&type=code
https://github.com/search?q=%22import+cx+from%22&type=code

But more importantly, we are currently using `cn` in our repository. I think we should keep `cn` or change it everywhere.

Reply:
Ok, no need to change it now then.